kinde_sdk/test/test_kinde_api_client.py

```
@@ -142,6 +148,86 @@ def mock_get_claim_side_effect(key):
+    def test_get_claim_token(self):
+        client = self._create_kinde_client(GrantType.AUTHORIZATION_CODE)
+        
+        result = client.get_claim_token({"access_token":{"org_code":"org123","permissions": ["read", "write", "delete"]}},"org_code")
+
+        expected_result = {
+            "name": "org_code",
+            "value": "org123"
+        }
+        self.assertEqual(result, expected_result)
+
+    def test_get_claim_token(self):
```

Ruff (0.8.2)

Redefinition of unused `test_get_claim_token` from line 174

(F811)